integrationTest/providers.json

```diff
@@ -58,6 +58,15 @@
     "api_key": "$DNSMADEEASY_API_KEY",
     "secret_key": "$DNSMADEEASY_SECRET_KEY"
   },
+  "EDGEDNS": {
```

Comment:
Would you be open to a name that includes Akamai?  Outsiders are more familiar with the name Akamai than EdgeDNS.  

We learned the hard way that these names are difficult to change.  As a result we have "GCLOUD" and "oracle" and "route53" which aren't optimal names.  I'd like to make sure we put a lot of thought into any new names.

Reply:
Done.  Now "akamaiedgedns".